packages/shared-base/src/Messages/Mask.ts

```diff
@@ -73,6 +73,7 @@ export interface CompositionDialogEvent {
     readonly options?: {
         target?: EncryptionTargetType
         startupPlugin?: string
+        startupPluginProps?: any
```

Comment:
Can we use a generic type stead?

Reply:
I think it will be very complicate. We might need to refactor plugin management, to make controlling plugin dialogs more easy.